src/components/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import Contacts from "./Contacts";
+import Logo from "./Logo";
+
+const Header = () => {
+  const [isBurgerToggled, setIsBurgerToggled] = useState(false);
+
+  const toggleBurgerHandler = () => {
+    setIsBurgerToggled(!isBurgerToggled);
+  };
+
+  return (
+    <nav className="header">
+      <Logo />
+      <ul className={`nav-links ${isBurgerToggled ? "nav-active" : ""}`}>
+        <li className={isBurgerToggled ? "nav-links-active" : ""}>
```

Comment:
Can be extracted as a separate and reusable component
<li>
<a href="#"></a>
</li>